feature/onboarding/welcome/src/main/kotlin/app/k9mail/feature/onboarding/welcome/ui/WelcomeContent.kt

```diff
@@ -1,5 +1,6 @@
 package app.k9mail.feature.onboarding.welcome.ui
 
+
```

Comment:
I guess your using a different code format. Just run `./gradlew spotlessApply` to streamline your changes with our code format. If you update the branch to current main, Spotless is also way quicker.